modules/snap_community_charts/charts.php

```diff
@@ -43,34 +48,35 @@
 
 <div class="buttonset" id="unit_buttons"><input checked="checked" id="unit_standard" name="units" type="radio" value="standard" /><label for="unit_standard">°F</label> <input id="unit_metric" name="units" type="radio" value="metric" /><label for="unit_metric">°C</label></div>
 </div>
-</div>
 
-<div>
-<p>Emissions scenario</p>
+<div class="inline-controls pull-left" style='padding-top: 19px;'>
```

Comment:
Agreed! I forgot I had been playing around with moving that div and forgot to include it back into the CSS when I got the div where I wanted it.